src/test/java/com/github/freeclimbapi/SMSTenDLCBrandTest.java

```diff
@@ -35,761 +34,737 @@
-    public void toStringTypeTest() {
-      SMSTenDLCBrand test1 = new SMSTenDLCBrand();
-      test1.setAccountId("TS");
-      test1.setEntityType(SMSTenDLCBrand.EntityTypeEnum.PRIVATE_PROFIT);
-      test1.setCspId("TS");
-      test1.setBrandId("TS");
-      test1.setFirstName("TS");
-      test1.setLastName("TS");
-      test1.setDisplayName("TS");
-      test1.setCompanyName("TS");
-      test1.setEin("TS");
-      test1.setEinIssuingCountry("TS");
-      test1.setPhone("TS");
-      test1.setStreet("TS");
-      test1.setCity("TS");
-      test1.setState("TS");
-      test1.setPostalCode("TS");
-      test1.setCountry("TS");
-      test1.setEmail("TS");
-      test1.setStockSymbol("TS");
-      test1.setStockExchange(SMSTenDLCBrand.StockExchangeEnum.NONE);
-      test1.setIpAddress("TS");
-      test1.setWebsite("TS");
-      test1.setBrandRelationship(SMSTenDLCBrand.BrandRelationshipEnum.BASIC_ACCOUNT);
-      test1.setVertical("TS");
-      test1.setAltBusinessId("TS");
-      test1.setAltBusinessIdType(SMSTenDLCBrand.AltBusinessIdTypeEnum.NONE);
-      test1.setUniversalEin("TS");
-      test1.setReferenceId("TS");
-        Map<String, Object> testMap = new HashMap();
-      test1.setOptionalAttributes(testMap);
-      test1.setMock(true);
-      test1.setIdentityStatus(SMSTenDLCBrand.IdentityStatusEnum.SELF_DECLARED);
-      test1.setCreateDate(OffsetDateTime.parse("2022-07-05T15:17:05Z"));
-
-      String toString1 = test1.toString();
-      Assert.assertTrue(String.class.isInstance(toString1));
-    }
-
- }
\ No newline at end of file
+public class SMSTenDLCBrandTest {
+  private final SMSTenDLCBrand model = new SMSTenDLCBrand("TEST_STRING", "TEST_STRING", "TEST_STRING");
+
+  /**
+   * Test the property 'accountId'
+   */
+  @Test
+  public void accountIdTest() {
+    model.setAccountId("TEST_STRING");
+    Assert.assertEquals("TEST_STRING", model.getAccountId());
+
+  }
+
+  /**
+   * Test the property 'entityType'
+   */
+  @Test
+  public void entityTypeTest() {
+
+    model.setEntityType(SMSTenDLCBrand.EntityTypeEnum.PRIVATE_PROFIT);
+    Assert.assertEquals(model.getEntityType(), SMSTenDLCBrand.EntityTypeEnum.PRIVATE_PROFIT);
+
+    model.setEntityType(SMSTenDLCBrand.EntityTypeEnum.PUBLIC_PROFIT);
+    Assert.assertEquals(model.getEntityType(), SMSTenDLCBrand.EntityTypeEnum.PUBLIC_PROFIT);
+
+    model.setEntityType(SMSTenDLCBrand.EntityTypeEnum.NON_PROFIT);
+    Assert.assertEquals(model.getEntityType(), SMSTenDLCBrand.EntityTypeEnum.NON_PROFIT);
+
+    model.setEntityType(SMSTenDLCBrand.EntityTypeEnum.GOVERNMENT);
+    Assert.assertEquals(model.getEntityType(), SMSTenDLCBrand.EntityTypeEnum.GOVERNMENT);
+
+    model.setEntityType(SMSTenDLCBrand.EntityTypeEnum.SOLE_PROPRIETOR);
+    Assert.assertEquals(model.getEntityType(), SMSTenDLCBrand.EntityTypeEnum.SOLE_PROPRIETOR);
+
+  }
+
+  /**
+   * Test the property 'cspId'
+   */
+  @Test
+  public void cspIdTest() {
+
+    Assert.assertEquals("TEST_STRING", model.getCspId());
```

Comment:
Do these tests work since there isn't anything being set?

Reply:
Yes they do, I have a model where I set the values into that model, that way I can compare the getter to the actual value.